plugins/inputs/postgresql_extensible/postgresql_extensible.go

```diff
@@ -278,6 +299,13 @@ COLUMN:
 			continue
 		}
 
+		if col == p.Timestamp {
+			if v, ok := (*val).(time.Time); ok {
+				timestamp = v
+			}
```

Comment:
might want to consider string fields that need to be parsed as timestamps, but this could be handled later.